### PR TITLE
docs: fix typos found with cspell

### PR DESCRIPTION
**Pull request by jboutsicaris**
word list from using cspell:

[project-words.txt](https://github.com/Nexusoft/LLL-TAO/files/12178446/project-words.txt)

### CODE REVIEW

**Comment by NamecoinGithub** [comment]
Wow it's like no one reads the Code :(  

Who made all those Errors?

AI Coders will Win for Sure!

Nexus needs Github AI-Pilot Coders

**Comment by jboutsicaris** [comment]
@NamecoinGithub please let the adults work. I can teach you how to make a PR if you ask nicely. Thanks

File: docs/API/COMMANDS/NAMES.MD [hunk 1]
```diff
@@ -48,8 +48,8 @@ The following nouns are supported for this API command-set:
 
 [`name`] -  An object register containing a name object.  
 [`namespace`] - An object register containing a namespace object.  
-[`global`] - An object register which is recognised globally on the network.  
-[`local`] - An object register which is recognised only within the context of a user profile.  
+[`global`] - An object register which is recognized globally on the network.
```
Comment (NamecoinGithub) [inline docs/API/COMMANDS/NAMES.MD:51]: I think The Original is Correct usage!

File: docs/API/COMMANDS/NAMES.MD [hunk 2]
```diff
@@ -48,8 +48,8 @@ The following nouns are supported for this API command-set:
...
+[`local`] - An object register which is recognized only within the context of a user profile.
```
Comment (NamecoinGithub) [inline docs/API/COMMANDS/NAMES.MD:52]: I think The Original is Correct usage!

**Comment by NamecoinGithub** [comment]
I would Break this up into probably 10 Different Pull Requests, I would say this large of things has like 0% chance of being Merged as is!  

Thanks, hope to find you Well!

**Comment by jboutsicaris** [comment]
Nice trolling

**Comment by jboutsicaris** [comment]
I saw your AI bots. This project is a cult anyways. Have fun with your bots
and crypto

**Comment by NamecoinGithub** [comment]
Well,

I am unsure, what Offends you other than the Lack of a sense of Humor?

All "New" Technology is Driven by what people might "Call" a "Cult" if they want to make the mistake in it's character judgement!

But, what I think you mean by "Cult" instead of using Technology Enthusiast as is more common, but Cult is much more demeaning as you are implying "Demon" and "God" Cult statuses upon an Inanimate Object!

My AI work has nothing to do with Nexus, I am just a Volunteer, No one pays me anything and I don't ask for anything! I try to do my Best and that is it and NOT make mistakes, but mistakes happen, and 1 mistake I know that did NOT happen is you ever seeing "My AI BOTS"!

I wonder if your being serious at this point really! Are you that Paranoid in Life?

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by jboutsicaris** [comment]
As entertaining as your responses are, this is a closed PR and you are
obnoxious. Do not expect anything else to come from your behavior

**Pull request closed without merging**